### ISSUE TITLE
`pluginInfo` missing on track model in Lavalink 4.0.1

### ISSUE THREAD

**Issue by angelobreuer** [lavalink-devs/Lavalink#1006]
## Description

Hello, a few minutes ago, I have received a user report regarding failing track resolution (library: `Lavalink4NET`). The issue is only present after switching from v4.0.0 to v4.0.1. Lavalink v4.0.1 no longer returns the `pluginInfo` field that was present before (even when being empty). The presence (regardless of data being present or not) of that field is documented in the documentation for the REST API. 

The intention of this request is more to notify of the introduction of a possible breaking change. If this change is intended, please feel free to close this issue. Adapting the library is quite easy.

## To Reproduce
1. Use Lavalink v4.0.1
2. Send a HTTP GET request to `http://localhost:2333/v4/loadtracks?identifier=dQw4w9WgXcQ`

```json
{
	"loadType": "track",
	"data": {
		"encoded": "QAAA3wMAPFJpY2sgQXN0bGV5IC0gTmV2ZXIgR29ubmEgR2l2ZSBZb3UgVXAgKE9mZmljaWFsIE11c2ljIFZpZGVvKQALUmljayBBc3RsZXkAAAAAAAM8IAALZFF3NHc5V2dYY1EAAQAraHR0cHM6Ly93d3cueW91dHViZS5jb20vd2F0Y2g/dj1kUXc0dzlXZ1hjUQEAOmh0dHBzOi8vaS55dGltZy5jb20vdmlfd2VicC9kUXc0dzlXZ1hjUS9tYXhyZXNkZWZhdWx0LndlYnAAAAd5b3V0dWJlAAAAAAAAAAA=",
		"info": {
			"identifier": "dQw4w9WgXcQ",
			"isSeekable": true,
			"author": "Rick Astley",
			"length": 212000,
			"isStream": false,
			"position": 0,
			"title": "Rick Astley - Never Gonna Give You Up (Official Music Video)",
			"uri": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
			"sourceName": "youtube",
			"artworkUrl": "https://i.ytimg.com/vi_webp/dQw4w9WgXcQ/maxresdefault.webp",
			"isrc": null
		}
	}
}
```

## Expected behavior

The following is the response to the same request from Lavalink v4.0.0:

```json
{
	"loadType": "track",
	"data": {
		"encoded": "QAAA3wMAPFJpY2sgQXN0bGV5IC0gTmV2ZXIgR29ubmEgR2l2ZSBZb3UgVXAgKE9mZmljaWFsIE11c2ljIFZpZGVvKQALUmljayBBc3RsZXkAAAAAAAM8IAALZFF3NHc5V2dYY1EAAQAraHR0cHM6Ly93d3cueW91dHViZS5jb20vd2F0Y2g/dj1kUXc0dzlXZ1hjUQEAOmh0dHBzOi8vaS55dGltZy5jb20vdmlfd2VicC9kUXc0dzlXZ1hjUS9tYXhyZXNkZWZhdWx0LndlYnAAAAd5b3V0dWJlAAAAAAAAAAA=",
		"info": {
			"identifier": "dQw4w9WgXcQ",
			"isSeekable": true,
			"author": "Rick Astley",
			"length": 212000,
			"isStream": false,
			"position": 0,
			"title": "Rick Astley - Never Gonna Give You Up (Official Music Video)",
			"uri": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
			"sourceName": "youtube",
			"artworkUrl": "https://i.ytimg.com/vi_webp/dQw4w9WgXcQ/maxresdefault.webp",
			"isrc": null
		},
		"pluginInfo": {}
		"userData": {}
	}
}
```

```patch
{
	"loadType": "track",
	"data": {
		"encoded": "QAAA3wMAPFJpY2sgQXN0bGV5IC0gTmV2ZXIgR29ubmEgR2l2ZSBZb3UgVXAgKE9mZmljaWFsIE11c2ljIFZpZGVvKQALUmljayBBc3RsZXkAAAAAAAM8IAALZFF3NHc5V2dYY1EAAQAraHR0cHM6Ly93d3cueW91dHViZS5jb20vd2F0Y2g/dj1kUXc0dzlXZ1hjUQEAOmh0dHBzOi8vaS55dGltZy5jb20vdmlfd2VicC9kUXc0dzlXZ1hjUS9tYXhyZXNkZWZhdWx0LndlYnAAAAd5b3V0dWJlAAAAAAAAAAA=",
		"info": {
			"identifier": "dQw4w9WgXcQ",
			"isSeekable": true,
			"author": "Rick Astley",
			"length": 212000,
			"isStream": false,
			"position": 0,
			"title": "Rick Astley - Never Gonna Give You Up (Official Music Video)",
			"uri": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
			"sourceName": "youtube",
			"artworkUrl": "https://i.ytimg.com/vi_webp/dQw4w9WgXcQ/maxresdefault.webp",
			"isrc": null
-		}
+		},
+		"pluginInfo": {}
+		"userData": {}
	}
}
```

## Version info
**Client used:** 

Output of `java -version`:
```
java version "17.0.10" 2024-01-16 LTS
Java(TM) SE Runtime Environment (build 17.0.10+11-LTS-240)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.10+11-LTS-240, mixed mode, sharing)
```

Lavalink Logs:
```
From expected behavior:

       .   _                  _ _       _    __ _ _
      /\\ | | __ ___   ____ _| (_)_ __ | | __\ \ \ \
     ( ( )| |/ _` \ \ / / _` | | | '_ \| |/ / \ \ \ \
      \\/ | | (_| |\ V / (_| | | | | | |   <   ) ) ) )
       '  |_|\__,_| \_/ \__,_|_|_|_| |_|_|\_\ / / / /
    =========================================/_/_/_/

        Version:        4.0.0
        Build time:     03.12.2023 14:46:34 UTC
        Branch          HEAD
        Commit:         acc6069
        Commit time:    03.12.2023 14:45:17 UTC
        JVM:            17.0.10
        Lavaplayer      2.0.4

From current behavior:

       .   _                  _ _       _    __ _ _
      /\\ | | __ ___   ____ _| (_)_ __ | | __\ \ \ \
     ( ( )| |/ _` \ \ / / _` | | | '_ \| |/ / \ \ \ \
      \\/ | | (_| |\ V / (_| | | | | | |   <   ) ) ) )
       '  |_|\__,_| \_/ \__,_|_|_|_| |_|_|\_\ / / / /
    =========================================/_/_/_/

        Version:        4.0.1
        Build time:     05.02.2024 12:25:46 UTC
        Branch          HEAD
        Commit:         a527886
        Commit time:    05.02.2024 12:23:17 UTC
        JVM:            17.0.10
        Lavaplayer      2.1.0

```

**Comment by topi314** [comment]
this is being worked on, we will issue a new release with a fix soon

**Comment by angelobreuer** [comment]
This has been fixed in v4.0.2 and later
